runtime/environment-cpu.yml

```diff
@@ -31,10 +31,12 @@ dependencies:
   - tensorflow-cpu=2.10.0
   - tqdm=4.64.1
   - xgboost=1.7.1
+  - future=0.18.2
```

Comment:
Can you explain the purpose of including the future package? The runtime is running Python 3.9, so my understanding is that future package won't do anything useful.

Reply:
Modified the code to get rid of the use of future